src/models/tf_model.py

```diff
@@ -81,6 +81,10 @@ def __init__(self, config):
         tf.set_random_seed(config['seed'])
 
         super(TFModel, self).__init__(config)
+        if config['base_dir']:
+          self._summary_writer = tf.summary.FileWriter(config['base_dir'])
```

Comment:
done

Reply:
Instead of having `base_dir` as part of config file, would it be better to use the `checkpt_dir` argument from `train.py` and add that to config dynamically and use that to write tensorboard information to? This way we don't have to change `base_dir` location in .yaml file every time we are training a different model and just use the `checkpt_dir` argument directly?

Reply:
done